src/core/qgsogrutils.cpp

```diff
@@ -20,6 +20,7 @@
 #include "qgsfields.h"
 #include <QTextCodec>
 #include <QUuid>
+#include <QtDebug>
```

Comment:
It is needed when using qDebug()
* which is useful when testing a version that is not compiled with DEBUG